docs/getting_started.md

```diff
@@ -187,6 +187,21 @@ While setting up `Context`, you'll be able to set [which version of the Admin AP
     console.log('your app is now listening on port 3000');
   });
   ```
+  
+  You will also need to set up a basic router to be able to process requests:
+
+  ```ts
+  
+  app.get('/', async (req, res) => {
+    const currentSession = await Shopify.Utils.loadCurrentSession(req, res);
```

Comment:
We shouldn't be calling `loadCurrentSession` here, that won't work for embedded apps. Maybe we should fix that here? An extremely simple way of doing this with app bridge v2 would be to check for `host` in the URL, since this request is essentially unauthenticated - i.e. if it loads inside of the admin, we can let it through and require permissions when we need them.